elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -236,15 +246,23 @@ class PfeJumpLinksNav extends PFElement {
       return false;
     }
     if (
-      (this.has_slot("logo") || this.has_slot("link")) &&
-      !this.hasAttribute("pfe-c-horizontal")
+      (this.shadowRoot.querySelector("logo") ||
+        this.shadowRoot.querySelector("link")) &&
```

Comment:
I'm wondering about these selectors, they would match `<logo>` and `<link>` so how does this look?

```suggestion
      (this.shadowRoot.querySelector("[slot='logo']") ||
        this.shadowRoot.querySelector("[slot='link']")) &&
```

Reply:
I am confused about the slots though, looking at the element's code and the demo page, I don't see anything with `slot="logo"` or `slot="link"`.

I do see `slot="pfe-jump-links-nav__logo"`.

Reply:
Ah, I see it's printing a warning if it detects those slots, so maybe those are deprecated slot names.  So I think we're good!